_content/news/_posts/2018-10-19-kerstkampen.markdown

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title:  "Basketbal voor de jeugd tijdens de kerstperiode"
-cover: ![Jeugd](Website/_content/albums/2018-10-13-Hasselt-BT-G12B-Basket-Lummen-G12C/_normalized/IMG_6268.jpg)
+cover: ![Jeugd](/albums/2018-10-13-Hasselt-BT-G12B-Basket-Lummen-G12C/_normalized/IMG_6268.jpg)
```

Comment:
Hier moet het gewoon

`cover: /albums/2018-10-13-Hasselt-BT-G12B-Basket-Lummen-G12C/_normalized/IMG_6268.jpg`

zijn, dus **niet** de ![title](/path) notatie, dat werkt enkel in het markdown stuk hieronder